src/main/java/it/pagopa/pn/deliverypush/utils/Constants.java

```diff
@@ -0,0 +1,6 @@
+package it.pagopa.pn.deliverypush.utils;
+
+public class Constants {
```

Comment:
Piuttosto di creare questa constants, aggiungiam 2 property nell'application.property in modo che compaiano dentro all'oggetto di configurazione di pndeliverypushconfig.webhook, che dici?